src/XIVLauncher/Resources/Loc/xl/xl_no.json

```diff
@@ -416,7 +416,7 @@
     "description": "MainWindowViewModel.SetupLoc"
   },
   "LoginBoxNoThirdLogin": {
-    "message": "Start w/o Third-Party Plugins",
+    "message": "Start w/o Custom Repo Plugins",
```

Comment:
If you want to be funny and actually translate this line, it would be `Start uten egendefinerte depot plugins`

Reply:
Probably best to contribute translations directly to https://crowdin.com/project/ffxivquicklauncher